styles.css

```diff
@@ -91,6 +107,172 @@ img {
 }
 
 .social-icon {
-  color: #505f79;
   padding-right: 10px;
+  color: #505f79;
+}
+
+.work {
+  margin: 2rem;
+  display: grid;
+  grid-template-rows: repeat(4, 1fr);
+  grid-row-gap: 2rem;
+}
+
+.single-card {
```

Comment:
- [ ] It would be nice to align the content of each project card with `flex`. Flexbox is a really great way to layout content on your page and it helps with responsiveness as well. You can read more about it [here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) 👍

Reply:
Sorry, can't do that. The requirements of the project are as below:
In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.